src/services/GooglePlacesService.ts

```diff
@@ -87,7 +87,14 @@ class GooglePlacesService {
         },
       })
       .then((response: AxiosResponse) => {
-        const restaurantResults = response?.data?.result;
+        const restaurantData = response?.data;
+        const restaurantResults = restaurantData?.result;
```

Comment:
Sort of an unrelated note, 
https://github.com/ubclaunchpad/PromoPal-frontend/blob/f585ec71668c22d4753096e7dcba970e20e665f8/src/services/GooglePlacesService.ts#L73-L75
How come we need to cast it? the value type is already `RestaurantDetails` right?

Reply:
Also could we make `currRestaurants` private?

Reply:
^ I was looking at this and think it should be private if possible too.

Reply:
We need to cast it cause there's a type error that it could be undefined

I made `currRestaurants` public for testing purposes and if we wanted to access data directly anywhere on FE- but how come you guys wanted to make it private?

Reply:
Oh but I think if you cast it and it is undefined, it will still be undefined
![image](https://user-images.githubusercontent.com/49849754/108161183-5bf7a780-709f-11eb-90d7-6703131cd520.png)

Reply:
Are you suggesting to do another conditional check inside, to check that its not undefined?

Reply:
Wait, if the placeID is valid, this.currRestaurants.get(placeID) will always return a defined RestaurantDetails, so I don't think it's necessary to have another check. But yeah I see why you would need to cast it, because "get" doesn't know if the placeID's valid.

Reply:
Ah shoot forgot there is an if statement that checks the existence of the key. Sorry about that